addon/components/search-help-modal/component.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+    layout,
+    isOpen: false,
+    currentPath: Ember.computed(function() {
+        return Ember.getOwner(this).lookup('controller:application').currentPath;
```

Comment:
Nothing needs to be added to the consuming app's Application Controller.  currentPath is already there https://guides.emberjs.com/v1.10.0/understanding-ember/debugging/#toc_get-current-route-name-path.  Different pages where this search modal will be used have different path names.  The examples in the search modal link build a search query for you. I need to know the currentPath. I can also pass in currentPath:"discover" or similar, alternatively.